### PR TITLE
Add gltfUpAxis support

### PR DESCRIPTION
**Pull request by javagl**
Details discussed in https://github.com/CesiumGS/cesium-unreal/issues/315

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @javagl!

* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] [Cesium for Unreal Samples](https://github.com/CesiumGS/cesium-unreal-samples) works.

**Review by baothientran — COMMENTED** [review]
@javagl this looks good to me. I only has a minor suggestion

File: Source/CesiumRuntime/Private/CesiumGltfComponent.cpp [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #include "mikktspace.h"
 #include <glm/ext/matrix_transform.hpp>
 #include <glm/gtc/quaternion.hpp>
+#include <glm/gtx/string_cast.hpp>
```
[the remark this answers is not in the document]
Comment (javagl) [reply Source/CesiumRuntime/Private/CesiumGltfComponent.cpp]: Thanks, removed it (that was used for some debug output)

OT: The build is still running, but I'm pretty sure that some compiler will not like the `double==enum` comparison. But doing that "right" is remarkably tricky: https://stackoverflow.com/q/18195312 ... Let's see.

[the remark this answers is not in the document]
Comment (baothientran) [reply Source/CesiumRuntime/Private/CesiumGltfComponent.cpp]: ah that's right. I forget the json value stores int as double. But the link above seems to be about casting int to enum. We can get away with it by doing the opposite by casting enum to int. Something like, cast the double value to int first, then cast the enum to int too, and compare the two values together

**Comment by baothientran** [comment]
@javagl can you please also update CHANGE.md here and on cesium-native side too?

File: Source/CesiumRuntime/Private/CesiumGltfComponent.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (javagl) [reply Source/CesiumRuntime/Private/CesiumGltfComponent.cpp]: None of the compilers complained about the `double==enum` part (which surprised me a bit). I now added casts to `int` according to your comment, but am/was not sure about the "best practices" for such a case.

**Comment by javagl** [comment]
Thanks @baothientran  for the review. I have integrated the changes and the build is currently running.

**Pull request merged**